ccd-definition/FixedLists.json

```diff
@@ -32,8 +32,14 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "AuthorityFixedList",
-    "ListElementCode": "example",
-    "ListElement": "Example Local Authority"
+    "ListElementCode": "swansea",
```

Comment:
ListElementCode needs to follow tech plan codes https://tools.hmcts.net/confluence/display/RPL/Notify+on+Submission

Swansea is SA, Hillingdon is HN